x-pack/plugins/security_solution/public/timelines/components/timeline/pin/index.tsx

```diff
@@ -17,21 +19,24 @@ export const getPinIcon = (pinned: boolean): PinIcon => (pinned ? 'pinFilled' :
 interface Props {
   allowUnpinning: boolean;
   iconSize?: IconSize;
+  timelineType?: TimelineTypeLiteral;
   onClick?: () => void;
   pinned: boolean;
 }
 
 export const Pin = React.memo<Props>(
-  ({ allowUnpinning, iconSize = 'm', onClick = noop, pinned }) => (
-    <EuiButtonIcon
-      aria-label={pinned ? i18n.PINNED : i18n.UNPINNED}
-      data-test-subj="pin"
-      iconSize={iconSize}
-      iconType={getPinIcon(pinned)}
-      isDisabled={allowUnpinning ? false : true}
-      onClick={onClick}
-    />
-  )
+  ({ allowUnpinning, iconSize = 'm', onClick = noop, pinned, timelineType }) => {
+    const isTemplate = timelineType === TimelineType.template;
+    return isTemplate ? null : (
```

Comment:
I thought that we discussed that in the team meeting, and we agreed that we will disable it with a tooltip